<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1181</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="createFile">
     <attribute name="title">
      <string>Create File</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Mex Produce</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Domestic</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mex Direct</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Overseas</string>
       </property>
      </item>
     </widget>
     <widget class="QStackedWidget" name="stackedSelection">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>391</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageMP">
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="10" column="1">
          <widget class="QPushButton" name="mexP_ProduceInv_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="mexP_SaleConf">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Sale Conf</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QPushButton" name="mexP_SignedSale_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="mexP_Invoice_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="mexP_Phyto">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Phyto</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="mexP_Passing_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="mexP_ShipperConf">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Shipper Conf</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="mexP_SignedSale">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Signed Sale Conf.</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QPushButton" name="mexP_Payable_Shipper_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QPushButton" name="mexP_Payables_Carriers_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QCheckBox" name="mexP_Pedimento">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Pedimento</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="mexP_Spreadsheets_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QPushButton" name="mexP_Pedimento_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QCheckBox" name="mexP_Payable_Shipper">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Payables-Shipper</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="mexP_Phyto_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QCheckBox" name="mexP_ProduceInv">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Produce Inv. </string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="mexP_ShipperConf_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="mexP_Spreadsheets">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Spreadsheets</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="mexP_CarrierConf_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="mexP_Passing">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Passing</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="mexP_Invoice">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Invoice</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="mexP_CarrierConf">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Carrier Conf</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QCheckBox" name="mexP_Payables_Carriers">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Payables-Carriers</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="mexP_SaleConf_upload">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Uploaded?</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="11" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item row="17" column="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>240</y>
        <width>461</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>220</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Notes:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>1069</x>
        <y>510</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="POInput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>9</number>
      </property>
      <property name="placeholderText">
       <string>PO#</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PO#:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openFolder">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Folder</string>
      </property>
     </widget>
     <widget class="QLabel" name="WAEX_logo">
      <property name="geometry">
       <rect>
        <x>855</x>
        <y>5</y>
        <width>321</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="runArchiveCheck">
     <attribute name="title">
      <string>Run Archive Check</string>
     </attribute>
     <widget class="QLabel" name="WAEX_logo_2">
      <property name="geometry">
       <rect>
        <x>855</x>
        <y>5</y>
        <width>321</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1198</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
